bin/nanocld/migrations/2023-05-29-121948_custom_url_hook/up.sql

```diff
@@ -0,0 +1,10 @@
+-- Your SQL goes here
```

Comment:
This is already existing inside `bin/nanocld/migrations/2023-01-15-121652_resources`